backend/server/core/services/getTaskArchive.service.ts

```diff
@@ -11,30 +11,31 @@ import {StateItem} from "../enums/stateItem.enum";
 // type GetTaskArchive = (challengeId: number) => ArchiveItem[];
 
 export function getTaskArchive(
-    challengeId: number,
-    challenges: Challenge[]
+  challengeId: number,
+  challenges: Challenge[]
 ): ArchiveItem[] {
-    const challenge: Challenge = challenges.find(
-        (challenge) => challenge.id === challengeId
-    );
+  const challenge: Challenge = challenges.find(
+    (challenge) => challenge.id === challengeId
+  );
 
-    const timeBetween = new Date().getTime() - challenge.startDate.getTime();
-    const daysBetween = Math.floor(timeBetween / (1000 * 3600 * 24));
-    const passedTasks: Task[] = challenge.tasksOrder.slice(0, daysBetween);
+  const timeBetween = new Date().getTime() - challenge.startDate.getTime();
+  const daysBetween = Math.floor(timeBetween / (1000 * 3600 * 24));
+  const passedTasks: Task[] = challenge.tasksOrder.slice(0, daysBetween);
 
-    const archiveItems: ArchiveItem[] = passedTasks.map((passedTask, dayTask) => {
-            const date = new Date(Date.now() - (daysBetween - dayTask) * (3600 * 1000 * 24));
-            const taskStatus: Status = {
-                state: StateItem.SUCCESS,
-                updated: date,
-            };
-            const archiveItem: ArchiveItem = {
-                id: passedTask.id,
-                description: passedTask.description,
-                status: taskStatus,
-            };
-            return archiveItem;
-        }
+  const archiveItems: ArchiveItem[] = passedTasks.map((passedTask, dayTask) => {
```

Comment:
Redundant const (you can use `return archiveItems: ...`)
Good IDE like WebStorm will say you about this)